console/console-init/ui/src/modules/iot-device/components/MangeColumnModal/ManageColumnModal.tsx

```diff
@@ -56,6 +59,7 @@ const ManageColumnModal: React.FunctionComponent<IManageColumnModalProps> = ({
     const columns = manageColumnsOptions
       .filter((column: IManageCoulmnOption) => column.isChecked === true)
       .map((column: IManageCoulmnOption) => column.value);
+    setSortValue(undefined);
```

Comment:
try to use empty string instead of undefined

Reply:
sortValue is a object not a string so that's why i used undefined